evaluate_versions.py

```diff
@@ -218,13 +243,20 @@ def main():
 
     file_name = os.path.join(CUR_DIR, XLSX_RESULT_FILE_NAME)
     workbook = xlsxwriter.Workbook(file_name)
+    collect_id = 0
+
     for i in range(0, len(first_collects)):
         bench_name = os.path.basename((first_collects[i].split(".collect"))[0])
 
         first_blocks_inst_count, first_dyn_inst_count = summary_blocks(first_collects[i])
         second_blocks_inst_count, second_dyn_inst_count = summary_blocks(second_collects[i])
 
         diff_result = compare_blocks(first_blocks_inst_count, second_blocks_inst_count)
+
+        if "x264_r" in bench_name:
```

Comment:
I think we should not do it here. We already have `delete_needless_collects` function for that purpose